docs/integrations/sources/s3.md

```diff
@@ -191,6 +191,10 @@ You can find details on [here](https://arrow.apache.org/docs/python/generated/py
 
 The avro parser uses [fastavro](https://fastavro.readthedocs.io/en/latest/). Currently, no additional options are supported.
 
+### Json
+
+The Json parser uses pyarrow. For more detailed info, please refer to the [docs] (https://arrow.apache.org/docs/python/generated/pyarrow.json.read_json.html)
+
```

Comment:
would be good to also clarify here in the docs that it's JSONL rather than any JSON.

Reply:
done